optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -409,16 +410,19 @@ def _from_pretrained(
             "tokenizer_2": None,
             "tokenizer_3": None,
             "feature_extractor": None,
-            "image_encoder": None,
             "safety_checker": None,
+            "image_encoder": None,
         }
         for name in submodels.keys():
-            if kwargs.get(name) is not None:
+            if name in kwargs:
                 submodels[name] = kwargs.pop(name)
             elif config.get(name, (None, None))[0] is not None:
-                library_name, library_classes = config.get(name)
-                library = importlib.import_module(library_name)
-                class_obj = getattr(library, library_classes)
+                module_name, module_class = config.get(name)
+                if hasattr(pipelines, module_name):
+                    module = getattr(pipelines, module_name)
+                else:
+                    module = importlib.import_module(module_name)
```

Comment:
in diffusers sometimes they import entire modules from `pipelines` https://github.com/huggingface/diffusers/blob/13e8fdecda91e27e40b15fa8a8f456ade773e6eb/src/diffusers/loaders/single_file.py#L65C7-L71C21